ts/modules/networkgraph/layouts.ts

```diff
@@ -194,6 +197,8 @@ extend(
             this.integration =
                 H.networkgraphIntegrations[options.integration as any];
 
+            this.enableSimulation = options.enableSimulation;
```

Comment:
Missing TS definition ;)

Reply:
TS definition was created before (not in this specific commit) as it was used in draggable-nodes